spec/dummy/config/alchemy/elements.yml

```diff
@@ -83,32 +83,41 @@
   contents:
     - name: essence_text
       type: EssenceText
+      classes: essence_text_classes
```

Comment:
Please use the `settings` Hash for that.

Reply:
> Please use the `settings` Hash for that.

But isn't the `settings` hash more unique for each essence? This setting would be global so i think it would be better defined top level?

Reply:
Content and essence have a one to one relationship. The settings hash is available for all instances of that particular content/essence.